src/app/components/hmcts-global-header/hmcts-global-header.component.html

```diff
@@ -57,8 +58,8 @@
     <div class="hmcts-primary-navigation__nav">
       <nav class="hmcts-primary-navigation transparent-background" aria-label="Primary navigation">
         <ul class="hmcts-primary-navigation__list">
-          <li class="hmcts-primary-navigation__item" *ngFor="let item of (leftItems | async)" >
-            <a class="hmcts-primary-navigation__link" (click)="onEmitSubMenu(item)" [attr.aria-current]="item.active ? true: null" [routerLink]="item.href">{{item.text}}</a>
+          <li class="hmcts-primary-navigation__item" *ngFor="let item of (leftItems | async)">
+            <a class="hmcts-primary-navigation__link" (click)="onEmitSubMenu(item)" [attr.aria-current]="item.active ? true : null" [routerLink]="item.href">{{item.text}}</a>
```

Comment:
Recommend setting attr.aria-current to false as opposed to null (unless when set to null it does not render the attribute)

Reply:
Hi @paulhowes-HMCTS - I've changed the setting to `false` for both `attr.aria-current` instances, when `!item.active`.

Reply:
Great @Daniel-Lam thanks.